Makefile

```diff
@@ -73,9 +74,9 @@ bindings:
 .PHONY: mock
 mock:
 	go generate -run mockgen ./...
+	$(MAKE) format-go
```

Comment:
This was easier than figuring out how to tell goimports to ignore the mockgen files.